File: Serene/Serene.Script/Meeting/MeetingType/MeetingTypeDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingTypeId"), NameProperty("Name"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingType"), LocalTextPrefix("Meeting.MeetingType"), Service("Meeting/MeetingType")]
    public class MeetingTypeDialog : EntityDialog<MeetingTypeRow>
    {
    }
}

File: Serene/Serene.AspNetCore/Modules/Meeting/MeetingType/MeetingTypeDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingTypeDialog extends Serenity.EntityDialog<MeetingTypeRow, any> {
        protected getFormKey() { return MeetingTypeForm.formKey; }
        protected getIdProperty() { return MeetingTypeRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingTypeRow.localTextPrefix; }
        protected getNameProperty() { return MeetingTypeRow.nameProperty; }
        protected getService() { return MeetingTypeService.baseUrl; }

        protected form = new MeetingTypeForm(this.idPrefix);

    }
}